### PR TITLE
Update SKILL.tsv

### PR DESCRIPTION
**Pull request by Sunaries**
Minor addition and fix.

### CODE REVIEW

File: SKILL.tsv [hunk 1]
```diff
@@ -1599,7 +1599,7 @@ SKILL_20150317_001598	Can't move
 SKILL_20150317_001599	Fear
 SKILL_20150317_001600	While, after the falter, attack is low.
 SKILL_20150317_001601	$Bleeding
-SKILL_20150317_001602	$Inflicts bleeding damage every 1.5 seconds.
+SKILL_20150317_001602	$Inflicts bleeding damage every 1.5 second
```
Comment (terrapin47) [inline SKILL.tsv]: Hmm technically should be seconds.

Comment (Sunaries) [inline SKILL.tsv]: Will revert this change after the weekend.

**Comment by Sunaries** [comment]
Good to merge.

**Comment by Sunaries** [comment]
Good to merge.

**Comment by Sunaries** [comment]
Hmm, mine has merging conflicts...?

**Comment by imcgames** [comment]
Thank you :smile:
